### PR TITLE
atari800_libretro: fix build and bump version

### PR DESCRIPTION
**Pull request by kwyxz**
Now uses the latest libretro-common submodule

### CODE REVIEW

File: app-emulation/atari800_libretro/atari800_libretro-3.1.0_20200429.recipe [hunk 1]
```diff
@@ -27,6 +32,7 @@ REQUIRES="
 	"
 
 BUILD_REQUIRES="
+	git$secondaryArchSuffix
```
Comment (waddlesplash) [inline app-emulation/atari800_libretro/atari800_libretro-3.1.0_20200429.recipe:35]: Is this really needed?

Comment (kwyxz) [reply app-emulation/atari800_libretro/atari800_libretro-3.1.0_20200429.recipe:35]: Maybe not. I added it for tests but if does build without it. I'll remove it.

Comment (kwyxz) [reply app-emulation/atari800_libretro/atari800_libretro-3.1.0_20200429.recipe]: Removed in [#067d672](https://github.com/haikuports/haikuports/pull/4966/commits/067d672708ed5b492f8957d9229fa64fe76fa383)

**Comment by kwyxz** [comment]
Goddammit I botched it with the flycast change. I'll fix and recreate a PR.

**Pull request closed without merging**